README.md

```diff
@@ -181,6 +181,16 @@ If there is only one [payment method](https://docs.commercelayer.io/developers/v
 
 In case the order balance is zero — e.g. the customer is paying with a gift card or coupon code for the full amount of the order — this step doesn't appear.
 
+### Order subscription
+
+#### Recurring items
+
+When a line item includes a frequency, placing the order [creates an order subscription](https://docs.commercelayer.io/core/v/how-tos/placing-orders/subscriptions/generating-the-subscriptions). To activate the subscription, the payment source must be saved in the customer's wallet. Therefore, during checkout with a customer token, the payment source is automatically saved. However, if a guest is checking out, they will receive an alert indicating that the subscription will not renew successfully without a saved payment source.
+
+#### Target Order 
+
+If a customer's payment source has expired, been deleted, or if the order initiating the subscription was placed as a guest, the resulting target order will be `pending`. Upon placement, the payment source will be automatically saved in the customer's wallet, and the order subscription will be automatically updated with the new payment source.
```

Comment:
"If a customer's payment source has expired or been deleted, or if the order..."